unstract/connectors/src/unstract/connectors/databases/redshift/redshift.py

```diff
@@ -5,10 +5,11 @@
 import psycopg2
 from psycopg2.extensions import connection
 
+from unstract.connectors.databases.psycopg_handler import PsycoPgHandler
 from unstract.connectors.databases.unstract_db import UnstractDB
 
 
-class Redshift(UnstractDB):
+class Redshift(UnstractDB, PsycoPgHandler):
```

Comment:
I think It's better to place PsycoPgHandler is first base then UnstractDB.

Reply:
@muhammad-ali-e Since Redshift and Postgres will also need PsycoPgHandler, its better to keep it here itself

Reply:
Yes, But we have to check which one should first, like if we have same method in each base class, The child class will prefer to take first base class than the second.